test/integration/bad/reduce_sum/stanc.expected

```diff
@@ -440,4 +457,4 @@ Semantic error in 'bad_userdefined_reduce_sum_static.stan', line 2, column 7 to
      4:    }
    -------------------------------------------------
 
-Identifier 'reduce_sum_static' clashes with Stan Math library function.
+Identifier 'reduce_sum_static' clashes with a non-overloadable Stan Math library function.
```

Comment:
Are there _any_ overloadable stan math library functions? idt so? If not then I'd remove the non-overloadable part of this

Reply:
All non-variadic functions can be overloaded in this PR